kentikapi/internal/errors/errors.go

```diff
@@ -0,0 +1,104 @@
+package errors
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type Code int
+
+const (
+	AuthError          Code = 1
+	InvalidRequest     Code = 2
+	NotFound           Code = 3
+	RateLimitExhausted Code = 4
+	Temporary          Code = 5
+	Timeout            Code = 6
+	Unavailable        Code = 7
+	Unknown            Code = 8
+	InvalidResponse    Code = 9
+)
+
+type KentikError struct {
```

Comment:
```suggestion
type StatusError struct {
```
I know that I didn't pointed it out earlier, but I had more important comments to make. I think that adding _Kentik_ prefix to the type located in _kentikapi_ module is not useful. However, StatusError tells that the error contains status code, which describes the purpose of the error type.

Those renames shouldn't be hard using an IDE.

Reply:
Done